README.md

```diff
@@ -83,6 +83,7 @@ See also [Uploading a SARIF file to GitHub](https://docs.github.com/en/code-secu
 |`rulesets`  |yes|        |Comma separated list of ruleset names to use.|
 |`analyzeModifiedFilesOnly`|no|"true"|Instead of analyze all files under "sourcePath", only the files that have been touched in a pull request or push will be analyzed. This makes the analysis faster and helps especially bigger projects which gradually want to introduce PMD. This helps in enforcing that no new code violation is introduced.<br>Depending on the analyzed language, the results might be less accurate results. At the moment, this is not a problem, as PMD mostly analyzes each file individually, but that might change in the future.<br>If the change is very big, not all files might be analyzed. Currently the maximum number of modified files is 300.<br>Note: When using PMD as a code scanner in order to create "Code scanning alerts" on GitHub, all files should be analyzed in order to produce a complete picture of the project. Otherwise alerts might get closed too soon.|
 |`createGitHubAnnotations`|no|"true"|By default, all detected violations are added as annotations to the pull request. You can disable this by setting FALSE. This can be useful if you are using another tool for this purpose.|
+|`createGitHubComments`|no|"false"|This flag used when you want to see violations as comments with the following format: `{priorityIcon} ({priority}) - {fileName} (Line:{startLine})`|
```

Comment:
This new property also needs to be added in `action.yml`, so that it is actually usable.

Reply:
Many thanks for your comment @adangel.
The `createGitHubComments` property added to `action.yml`.